train.gemspec

```diff
@@ -41,4 +41,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "docker-api", "~> 1.26"
   spec.add_dependency "google-api-client", ">= 0.23.9", "< 0.35.0"
   spec.add_dependency "googleauth", ">= 0.6.6", "< 0.11.0"
+  spec.add_dependency "dotenv", "~> 2.7.6"
+  spec.add_dependency "facets", "~> 3.1.0"
```

Comment:
Is this something we really need or are we just pulling these in so we can use some helpers? Train gets installed in a lot of places and this doesn't seem like a must have for implemented azure support.

Reply:
Working on removing `facets` and `dotenv`. However, `ipaddr` is essential to make security rules comparison work properly. It is provided in the AWS resource pack.